the-collab-lab/smart-shopping-list-deprecated
# Add Cypress to project template for e2e testing

## Description

Going forward we need to shave some of the time commitment of our smart shopping list project. We've decided to make testing optional but will still strongly encourage participants to practice writing tests. 

This commit adds Cypress to the project for end-to-end testing. We've left all the Cypress boilerplate directories, example tests, and documentation so that future participants can poke around and learn from those resources. 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|   ✓ | :sparkles: New feature     |
|  ✓  | :link: Update dependencies |

## Testing Steps / QA Criteria

* Run `cypress open` to open the Cypress Test Runner in interactive mode. From there you can run the example tests to see it in action. This is likely the first interaction our participants will have with Cypress in this project.
